app/build.gradle

```diff
@@ -27,7 +27,7 @@ android {
         versionCode parent.ext.androidVersionCode
         versionName parent.ext.androidVersionName
 
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
```

Comment:
I don't very much like the incongruence added between `'` and `"`. We need `"` for the strings that require a variable, so why not leaving all the strings like that? Both are equivalent.